addons/website_livechat/controllers/chatbot.py

```diff
@@ -57,9 +57,10 @@ def chatbot_test_script(self, chatbot_script):
         chatbot_script._post_welcome_steps(discuss_channel)
         store = Store()
         request.env["res.users"]._init_store_data(store)
+        store.add(chatbot_script)
         return request.render("im_livechat.chatbot_test_script_page", {
             'server_url': chatbot_script.get_base_url(),
             'channel_data': {'id': discuss_channel.id, 'model': 'discuss.channel'},
-            'chatbot_data': chatbot_script._format_for_frontend(),
+            'chatbot_data': chatbot_script.id,
```

Comment:
It might be cleaner to rename `chatbot_data` into `chatbot_script_id` in this case.
Or even better if we can just give `chatbot_script` to template and let it use `.id` or any other field it needs.